include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -1487,7 +1487,7 @@ namespace internal
     }
 
     bool hp_functionality_enabled = false;
-    for (const auto &dh : dof_handler)
+    for (const auto &dh : dof_handler.cell_iterators())
```

Comment:
`dof_handler` has type `const std::vector<SmartPointer<const DoFHandler<dim>>> &dof_handler`, right? So this should really not change.

Reply:
Oops! You're correct of course!